.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Can you squash this into one, commit, add your SoB and use the correct format for subject.